Assignment.java

```diff
@@ -0,0 +1,71 @@
+package package1;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class Assignment {
+	
+	WebDriver driver;
+
+	@Before
+	public void setupMethod() {
+		 System.setProperty("webdriver.chrome.driver","C:\\jar files\\chromedriver\\chromedriver.exe");
+		 driver = new ChromeDriver();
+		 driver.manage().window().maximize();
+	}
+		
+	@Test 
+	public void scenario2() {
+		
+		driver.get("https://www.traderev.com/en-ca/");
+		driver.findElement(By.linkText("Careers")).click();
+		ArrayList<String> tabs = new ArrayList<>(driver.getWindowHandles());
+		driver.switchTo().window(tabs.get(1));
```

Comment:
you are not validating anything here or you are missing this, Validate whether Canadian job site is displayed properly